src/transformers/models/omdet_turbo/processing_omdet_turbo.py

```diff
@@ -55,11 +69,23 @@ class OmDetTurboProcessorKwargs(ProcessingKwargs, total=False):
     }
 
 
-if is_torch_available():
-    import torch
+class _dict_with_warning(dict):
```

Comment:
let's use camel casing please!

Reply:
Fixed in https://github.com/huggingface/transformers/pull/34937/commits/2d7a6b24528d14c11bd6a3cd0b7b0a377374ab24